omerointegration/build.gradle

```diff
@@ -14,6 +14,9 @@ repositories {
 
 //excludes based on: https://github.com/glencoesoftware/omero-ms-core/blob/master/build.gradle
 dependencies {
+    external "com.google.protobuf:protobuf-java:${googleProtocolBufVersion}"
+    external("org.apache.commons:commons-collections4:${commonsCollections4Version}")
```

Comment:
I think the better way to do this, since these two jars are not actually used in the compilation (and we've got a slew of other jar files not accounted for in the jars.txt for this module) is to force the versions for these jars up in the root-level `build.gradle`.